src/inspector_socket.cc

```diff
@@ -559,21 +559,22 @@ static void init_handshake(InspectorSocket* inspector) {
   settings->on_url = path_cb;
 }
 
-int inspector_accept(uv_stream_t* server, InspectorSocket* inspector,
+// XXX(sam) shouldn't all these snake_case functions be CamelCase?
```

Comment:
This class was a plain C as it was mimicking libuv conventions. Need to become proper C++ at some point...